.dockerignore

```diff
@@ -0,0 +1,2 @@
+/target
+.envrc
```

Comment:
We don't need these things in the Docker conatiner for deploys so add these to the ignore file to make builds faster!

(The target dir gets big do adding it is slow and doesn't help at all, and the envrc file is local secrets we don't want in prod!)